{"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"Error.localizedDescription"},{"type":"text","text":"."}],"metadata":{"title":"localizedDescription","symbolKind":"property","externalID":"s:s5ErrorP10FoundationE20localizedDescriptionSSvp::SYNTHESIZED::s:10SwiftKyber9ExceptionO","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"localizedDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"modules":[{"name":"SwiftKyber","relatedModules":["Swift"]}],"platforms":[{"deprecated":false,"name":"iOS","beta":false,"introducedAt":"8.0","unavailable":false},{"name":"macOS","deprecated":false,"introducedAt":"10.10","unavailable":false,"beta":false},{"deprecated":false,"introducedAt":"9.0","beta":false,"unavailable":false,"name":"tvOS"},{"deprecated":false,"introducedAt":"2.0","name":"watchOS","unavailable":false,"beta":false}],"roleHeading":"Instance Property","extendedModule":"Swift"},"identifier":{"url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/localizeddescription"]}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"localizedDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}],"kind":"declarations"}],"schemaVersion":{"minor":3,"major":0,"patch":0},"sections":[],"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations"]]},"kind":"symbol","references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"abstract":[{"type":"text","text":"The Kyber exceptions"}],"title":"Exception","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","type":"topic","role":"symbol","url":"\/documentation\/swiftkyber\/exception","navigatorTitle":[{"text":"Exception","kind":"identifier"}],"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Exception"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/Error-Implementations":{"identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations","abstract":[],"kind":"article","type":"topic","role":"collectionGroup","title":"Error Implementations","url":"\/documentation\/swiftkyber\/exception\/error-implementations"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/localizedDescription":{"abstract":[],"title":"localizedDescription","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","type":"topic","role":"symbol","url":"\/documentation\/swiftkyber\/exception\/localizeddescription","kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"title":"SwiftKyber","abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"type":"topic","kind":"symbol","url":"\/documentation\/swiftkyber","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","role":"collection"}}}